bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/thing/sensor-thing-types.xml

```diff
@@ -111,6 +118,7 @@
 			<bridge-type-ref id="deconz"/>
 		</supported-bridge-type-refs>
 		<label>Fire Sensor</label>
+		<semantic-equipment-tag>AlarmDevice</semantic-equipment-tag>
```

Comment:
I'm not sure, is it an `AlarmDevice` or a `Sensor` (since there is no `FireSensor` currently)? I'm not sure what a fire sensor is compared to a smoke detector (`SmokeDetector`).

Reply:
Yeah we really need 

- Fire detector (parent)
- Smoke detector (child of Fire detector)
- Heat detector (child of Fire detector)

I will add to https://github.com/openhab/openhab-core/pull/4708